instrumentation/spring/spring-boot-autoconfigure/src/main/java/io/opentelemetry/instrumentation/spring/autoconfigure/propagators/PropagationType.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.spring.autoconfigure.propagators;
+
+/** Types of supported propagators. */
+public enum PropagationType {
+
+  /** B3 Single */
+  b3,
```

Comment:
We follow the [Google Style Guide](https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names) that recommends using CONSTANT_CASE for enums. 
Is it possible to use some kind of converter for those values? If not, I guess using plain strings (like `"b3"`) without an enum would be fine too.

Reply:
I think using just strings would make sense, thanks